internal/pkg/cosign/fulcio/fulcioroots/fulcioroots.go

```diff
@@ -0,0 +1,100 @@
+//
+// Copyright 2021 The Sigstore Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package fulcioroots
+
+import (
+	"bytes"
+	"crypto/x509"
+	"fmt"
+	"os"
+	"sync"
+
+	"github.com/sigstore/sigstore/pkg/cryptoutils"
+	"github.com/sigstore/sigstore/pkg/fulcioroots"
+)
+
+var (
+	rootsOnce        sync.Once
+	roots            *x509.CertPool
+	intermediates    *x509.CertPool
+	singletonRootErr error
+)
+
+const altRoot = "SIGSTORE_ROOT_FILE"
+
+// Get returns the Fulcio root certificate.
+//
+// If the SIGSTORE_ROOT_FILE environment variable is set, the root config found
+// there will be used instead of the normal Fulcio roots.
+//
+// If this behavior is not needed, use
```

Comment:
nit: not clear whether "this behavior" refers to "returns the Fulcio root certificate" or the bit about the env var.

Reply:
+1, would simply clarify that these methods support fetching from an environment variable too